Server/mods/deathmatch/logic/lua/CLuaArguments.cpp

```diff
@@ -611,46 +611,51 @@ json_object* CLuaArguments::WriteTableToJSONObject(bool bSerialize, CFastHashMap
 
     pKnownTables->insert(std::make_pair(this, pKnownTables->size()));
```

Comment:
Why not `{this, pKnownTables->size()}`?

Reply:
I left this code as-is in my first commit, changed it now with the second commit